### PR TITLE
[NEW] Understanding Common Cloud Manager Errors

### PR DESCRIPTION
**Pull request by rsyracuse**

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 3410f2c0a689cdf017ca3d773b8a7689d2eb5326

https://deploy-preview-3063--nostalgic-ptolemy-b01ab8.netlify.com

File: docs/quick-answers/linode-platform/an-overview-of-common-cloud-manager-errors/index.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+published: 2019-02-22
+modified_by:
+  name: Linode
+title: "An Overview of Common Cloud Manager Errors"
+h1_title: "Understanding Common Cloud Manager Errors"
+contributor:
+  name: Linode
+---
+
+Have you ever encountered an error message while navigating the [Cloud Manager](https://cloud.linode.com) and wanted to know what caused the error and what to do next? This guide will cover some common error messages, their common causes, suggestions on your best paths forward.
```
Comment (andystevensname) [inline docs/quick-answers/linode-platform/an-overview-of-common-cloud-manager-errors/index.md]: We probably don't need the first sentence. "This guide will cover some common error messages found in the Cloud Manager(link)..."

Also need the word "and" before "suggestions."

File: docs/quick-answers/linode-platform/an-overview-of-common-cloud-manager-errors/index.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+**Cause:** The above error means that there is an issue retrieving your Linodes from our backend servers.
+
+**Next Steps:**
+
+- As a first step, you should check our status page at [status.linode.com](https://status.linode.com) to ensure that there are no current issues that may be effecting your Linodes or Linode service.
+
+  - If you see an issue listed, you can rest assured that Linode is investigating it and will work to bring your services back online as quickly as possible.
+
+  - If the status page shows that everything is all clear, then the next step is to attempt to clear your web browser cache, as this can sometimes cause an issue in loading resources. The method for completing this process varies between browsers. However, it usually involves opening your browser's full history and finding an option to clear it.
```
Comment (andystevensname) [inline docs/quick-answers/linode-platform/an-overview-of-common-cloud-manager-errors/index.md]: "If the status pages shows that there are no known issues currently effecting service,..."

File: docs/quick-answers/linode-platform/an-overview-of-common-cloud-manager-errors/index.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+**Message:** *"Account Limit Reached. Please open a support ticket."*
+
+**Cause:** If you see the message above, then you'll be unable to create a resource due to a limit currently set on your account.
+
+**Next Steps:**
+
+- Linode's [Support Team](https://www.linode.com/docs/platform/billing-and-support/support/) will be able to help to either increase this limit, or to provide insight into why the limit may be in place.
+
+- The reasons behind these limits can vary, though, in most cases are related to default resource limits set on your account.
```
Comment (andystevensname) [inline docs/quick-answers/linode-platform/an-overview-of-common-cloud-manager-errors/index.md]: The reasons behind these limits can vary. In most cases **they** are related to default resource limits set on your account.

**Pull request merged**